tutorial_app/views.py

```diff
@@ -72,9 +80,11 @@ def get(self, request):
         :param request:
         :return:
         """
+        # how are you getting the request.user?
```

Comment:
After Authentication, it is providing the current user by "request.user".

Reply:
:+1: